resources/views/branchOffice/edit.blade.php

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Branch office form</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <style>
+        .alert {
+            margin-top: 10px;
+        }
+    </style>
+</head>
+<body>
+@extends('partials.navBar');
+
+<div class="container mt-5">
+    <h1>Enter information about the branch office</h1>
+    <div class="col-md-12">
+        <form
+            action="{{ isset($filteredData['uuid']) ? '/branchOffice/edit/' . $filteredData['uuid'] : '/branchOffice/create' }}"
+            method="POST">
+            @csrf
+            @if(isset($filteredData['name']))
```

Comment:
Simplify

Reply:
I don't know how should I simplify this.